src/modm/platform/extint/rp/module.md

```diff
@@ -0,0 +1,45 @@
+# External Interrupt Handler
+
+This driver provides an API for configuring all IRQ lines via register access.
+Note that you need to pass a mask, which allows you to configure multiple IRQ
+lines at once.
+
+```cpp
+GpioIntHandler::enable();
+
+// Powers on the LED on the low->high transition, and off on high->low.
+GpioInput0::setInput();
+GpioIntHandler::connect<GpioInput0>(Gpio::InputTrigger::BothEdges,
+									[](Gpio::InputTrigger triggers) {
+										Led::set(!!(triggers & Gpio::InputTrigger::RisingEdge));
+									});
+
+// Toggles LED each time gpio input is at the high level.
+GpioInput1::setInput(Gpio::InputType::PullDown);
+GpioIntHandler::connect<GpioInput1>(Gpio::InputTrigger::HighLevel,
+									[](Gpio::InputTrigger) { Led::toggle(); });
+```
+
+## Multicore mode
+
+Each core can register callbacks in the same IntHandler,
+but for the different pins (current implementation's constraint).
+
+Also, enable/disable and connect/disconnect calls
+affect the NVIC of the executing core only.
+
+## Callbacks
+
+The callback is implemented using `modm::inplace_function`, therefore uses no
+heap, but has a fixed storage size of `sizeof(void*)` by default.
+You can increase this storage size in your `project.xml` options:
+
+```xml
+<library>
+  <options>
+    <option name="modm:platform:irq:handler.storage">16</option>
+    <!-- or -->
+    <option name="modm:platform:irq:handler.storage">sizeof(uint64_t)</option>
```

Comment:
We try to not do this to enable people to customize small things like this in modm even after lbuild generation.